klone.yaml

```
@@ -47,3 +47,9 @@ targets:
       repo_ref: main
       repo_hash: 442cb8fe628deafa0b95c5b5f04576de7e5d84f1
       repo_path: modules/tools
+  make/_shared_new:
+    - folder_name: helm
+      repo_url: https://github.com/cert-manager/makefile-modules.git
+      repo_ref: main
+      repo_hash: ca4085279ab425a3e8503a5e227e94c6869ab31e
```

```suggestion
      repo_hash: 8187c6497881beb7829253c3467002b6cb5c5fb7
```

To fix the missing trailing newlines.